cpp/src/slice2swift/msbuild/slice2swift.vcxproj

```diff
@@ -20,7 +20,8 @@
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <ProjectGuid>{8D237BBB-506A-4CC1-8C81-BDC2787DA88B}</ProjectGuid>
-    <RootNamespace>slice2cs</RootNamespace>
+    <RootNamespace>slice2swift</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```

Comment:
All the other compilers had this platform version. I guess the `slice2swift` was lagging behind because we forgot it existed.